backend/gateway/gateway.go

```diff
@@ -134,11 +135,17 @@ func RunWithConfig(f *Flags, cfg *gatewayv1.Config, cf *ComponentFactory, assets
 		resolver.Registry[resolverCfg.Name] = res
 	}
 
+	accesslog, err := accesslog.New(cfg.Gateway.Accesslog, logger, scope)
```

Comment:
let's wrap this is in
```go
if cfg.Gateway.Accesslog != nil
```
so it's not enabled by default logging everything

Reply:
Oh I thought we were going to default to logging everything.
So if a user wants to enable logging for all the codes they would have to list all the codes in the config?

Reply:
If that's the case then should we have a default config in place in the backend clutch-config? Something like log all 5xx and 4xx?

Reply:
with nil check to enable access logging (and logging everything) they would pass an empty dict for the key,
`accesslog: {}`

default 5xx logging would also acceptable behavior

Reply:
Done :) and added default 5xx logging https://github.com/lyft/clutch/pull/804/files#diff-414d3d9d7c6a8dc5868d9121ddb15d30d16564a3a459c66f455129970e5c7ee1R5